ophyd/areadetector/plugins.py

```diff
@@ -67,6 +67,7 @@ def __init__(self, *args, **kwargs):
                                           self._plugin_type,
                                           self.plugin_type.get(), self.prefix))
 
+        self._enable_on_stage=True
```

Comment:
For PEP8 compliance, there should be spaces around the `=` sign when it's used for variable assignment -- here and a couple places below.